packages/devtools_app/lib/src/service/service_extension_manager.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe, unnecessary_null_comparison
+// ignore_for_file: unnecessary_null_comparison
```

Comment:
same question

Reply:
Addressed in https://github.com/flutter/devtools/pull/4011